frontend/public/components/operator-hub/operator-hub-items.tsx

```diff
@@ -111,11 +106,15 @@ const sortFilterValues = (values, field) => {
   }
 
   if (field === 'providerType') {
-    sorter = providerTypeSort;
+    sorter = ProviderType;
```

Comment:
You would still need the sort function as before. Just check the enum value in the `switch` block instead of the string.